docs/reference/contrib/redirects.md

```diff
@@ -93,3 +93,20 @@ Options:
 
     .. automethod:: add_redirect
 ```
+## API
+
+You can create an API endpoint to retrieve redirects by path.
+
+See the [](api_v2_configuration) documentation on how to configure the Wagtail API. 
+
+Add the following code to add the redirects endpoint:
+
+```python
+from wagtail.contrib.redirects.api import RedirectsAPIViewSet
+
+api_router.register_endpoint('redirects', RedirectsAPIViewSet)
+```
+
+With this configuration, redirects will be available at `/api/v2/redirects/find/?html_path=<path>`.
+
+This will return either a `200` response with the redirects detail, or a `404` not found response.
```

Comment:
I will push a refinement here that just adds a ref at the top of this section and differentiates between simple listings and the `find` approach.